features/shared/continuing_after_success.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Show clear error if trying to continue after executing a successful command
+
+  As a developer accidentally trying to continue a command after it completed successfully
+  I should see a clear error that the command cannot be continued
+  So that I don't experience any cryptic errors.
```

Comment:
I think @ricmatsui's original sentence makes more sense. Maybe in addition to to *cryptic* -- *irrelevant*?